Yosemite/Yosemite/Stores/OrderStore.swift

```diff
@@ -354,6 +358,10 @@ private extension OrderStore {
         remote.updateOrder(from: siteID, order: order, fields: fields) { [weak self] result in
             switch result {
             case .success(let order):
+                // Auto-draft orders are temporary and should not be stored
+                guard order.status != .autoDraft else {
+                    return onCompletion(result)
+                }
```

Comment:
I wonder if we should delete the order in case it is already stored 🤔

Reply:
I thought about that and it would be more thorough, but I also think it's overkill. Any order that started with `auto-draft` status won't be stored when it's created, and orders that exist already with a different status shouldn't be updated to have `auto-draft` status. (If we're worried about the second scenario possibly happening, I think it would make more sense to add tests against that scenario rather than trying to handle it here, since it shouldn't be possible.)